base/math.jl

```diff
@@ -620,6 +620,42 @@ function frexp{T<:AbstractFloat}(x::T)
 end
 
 """
+    rem(x, y, r::RoundingMode)
+
+Compute the remainder of `x` after integer division by `y`, with the quotient rounded
+according to the rounding mode `r`. In other words, the quantity
+
+    x - y*round(x/y,r)
+
+without any intermediate rounding.
+
+- if `r == RoundNearest`, then the result is exact, and in the interval
+  `[-abs(y)/2, abs(y)/2]`.
+
+- if `r == RoundToZero` (default), then the result is exact, and in the interval
+  `[0,abs(y))` if `x` is positive, or `(abs(y),0]` otherwise.
```

Comment:
maybe math-mode (double backtick) the intervals since that isn't Julia notation?